hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/TestdataFactory.java

```diff
@@ -620,27 +587,42 @@ public Target createTarget() {
      */
     public Target createTarget(final String controllerId) {
         final Target target = targetManagement.create(entityFactory.target().create().controllerId(controllerId));
+       assertTargetProperlyCreated(target);
+        return target;
+    }
+
+    /**
+     * @param targetName
+     *            name of the target
+     * @return persisted {@link Target}
+     */
+    public Target createTargetWithName(final String targetName) {
```

Comment:
add targetName parameter to `createTarget` method  and use overloading

Reply:
Done.